utils/Framework/Node/StatemineNode.ts

```diff
@@ -0,0 +1,84 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+import {
+  AssetSpec,
+  ChainId,
+  ChainSpec,
+  ChainSpecs,
+  TRANSFER_INSTRUCTIONS,
+} from "../../ChainSpecs";
+import { User } from "../../User";
+
+export class StatemineNode {
+  api: ApiPromise;
+  chain: ChainSpec;
+
+  xTokenTransfer(
+    toChain: ChainId,
+    assetId: AssetSpec,
+    amount: BN,
+    toUser: User
+  ): any {
+    expect(ChainSpecs.has(toChain));
```

Comment:
expects should be only for test expectations, perhaps we should use hee "asserts" ? 
I can fix them.

Reply:
https://blog.logrocket.com/assertion-functions-typescript/